trin-storage/src/versioned/id_indexed_v1/sql.rs

```diff
@@ -0,0 +1,96 @@
+use crate::versioned::ContentType;
+
+/// The name of the sql table. The `ii1` stands for `id_indexed_v1`.
+fn table_name(content_type: &ContentType) -> String {
+    format!("ii1_{content_type}")
+}
+
+pub fn create_table(content_type: &ContentType) -> String {
+    format!(
+        "
+        CREATE TABLE IF NOT EXISTS {0} (
+            content_id BLOB PRIMARY KEY,
+            content_key BLOB NOT NULL,
+            content_value BLOB NOT NULL,
+            distance_short INTEGER NOT NULL,
+            content_size INTEGER NOT NULL
+        );
+        CREATE INDEX IF NOT EXISTS {0}_distance_short_idx ON {0} (distance_short, content_id);
```

Comment:
I think it's different than that one, because here we don't have to iterate over all entries (we just select first/last X). But maybe I'm wrong...

I looked more into what queries would use it and how it affects them:
- the `lookup_farthest` would benefit from having `content_id`, but I think not to the great extend
- the `delete_farthest` would probably affected in a negative way, but I'm not sure by what amount (probably more than `lookup_farthest` would benefit)

So I will remove `content_id` for now, and later I might add an index that has both columns (once I do some experimentation).